frontend/src/main/java/puf/frisbee/frontend/viewmodel/GameViewModel.java

```diff
@@ -173,11 +177,31 @@ public void handle(long now) {
 				int characterSpeed = gameModel.getCharacterSpeed();
 				int gravity = gameModel.getGravity();
 
+				// BEGIN SOCKET COMMUNICATION
+				SocketClientFactory socketClientFactory = new SocketClientFactory();
```

Comment:
Von meinem Verständnis her müsste das doch in der `ViewModelFactory` passieren? Und dann wird der SocketClient hier einfach wie z.B. das GameModel übergeben? Und dann ruft man hier nur die SocketClient Methoden auf... alternativ könnte man es noch eine Ebene höher schieben und im Model die Client Connection machen (haben wir ja für unsere Requests auch so) und ein `updatePosition` in einem CharacterModel aufrufen, das dann das Ganze an den Server weiterschiebt?